src/views/cards/cards.jsx

```diff
@@ -52,7 +52,7 @@ var Cards = injectIntl(React.createClass({
                                     <FormattedMessage id='cards.viewCard' />
                                     {(
                                         formattedLinks['cards.starterLink'] === englishLinks['cards.starterLink'] &&
-                                        locale !== 'en'
+                                        locale.indexOf('en') === -1
```

Comment:
Yeah, was mostly thinking about it being robust for the future even though it works with the languages we have now.